assets/ajax-cart.js.liquid

```diff
@@ -381,260 +323,6 @@ var ajaxifyShopify = (function(module, $) {
-        template = Handlebars.compile(source),
-        data = {
-          wrapperClass: $wrapperClass
-        };
-
-    // Append drawer (defaults to body)
-    $prependDrawerTo.prepend(template(data));
-
-    // Drawer selectors
-    $drawerContainer = $('#AjaxifyDrawer');
-    $cartContainer   = $('#AjaxifyCart');
-    $drawerCaret     = $('.ajaxcart__caret > span');
-
-    // Toggle drawer with cart button
-    setToggleButtons();
-
-    // Position caret and size drawer on resize if drawer is visible
-    var timeout;
-    $(window).resize(function() {
-      clearTimeout(timeout);
-      timeout = setTimeout(function(){
-        if ($drawerContainer.hasClass('is-visible')) {
-          positionCaret();
-          sizeDrawer();
-        }
-      }, 500);
-    });
-
-    // Position the caret the first time
-    positionCaret();
-
-    // Position the caret
-    function positionCaret() {
-      if ($toggleCartButton.offset()) {
-        // Get the position of the toggle button to align the caret with
-        var togglePos = $toggleCartButton.offset(),
-            toggleWidth = $toggleCartButton.outerWidth(),
-            toggleMiddle = togglePos.left + toggleWidth/2;
-
-        $drawerCaret.css('left', toggleMiddle + 'px');
-      }
-    }
-  };
-
-  showDrawer = function (toggle) {
-    // If we're toggling with the flip method, use a special callback
-    if (settings.method == 'flip') {
-      Shopify.getCart(flipCartUpdateCallback);
-    }
-    // opening the drawer for the first time
-    else if (!cartInit && toggle) {
-      Shopify.getCart(cartUpdateCallback);
-    }
-    // simple toggle? just size it
-    else if (cartInit && toggle) {
-      sizeDrawer();
-    }
-
-    // Show the drawer
-    $drawerContainer.addClass('is-visible');
-
-    scrollTop();
-
-    toggleCallback({
-      'is_visible': true
-    });
-  };
-
-  hideDrawer = function () {
-    $drawerContainer.removeAttr('style').removeClass('is-visible');
-    scrollTop();
-    toggleCallback({
-      'is_visible': false
-    });
-  };
-
-  sizeDrawer = function ($empty) {
-    if ($empty) {
-      $drawerContainer.css('height', '0px');
-    } else {
-      $drawerHeight = $cartContainer.outerHeight();
-      $('.cart__row img').css('width', 'auto'); // fix Chrome image size bug
-      $drawerContainer.css('height',  $drawerHeight + 'px');
-    }
-  };
-
-  loadCartImages = function () {
-    // Size cart once all images are loaded
-    var cartImages = $('img', $cartContainer),
-        count = cartImages.length,
-        index = 0;
-
-    cartImages.on('load', function() {
-      index++;
-
-      if (index==count) {
-        switch (settings.method) {
-          case 'modal':
-            sizeModal();
-            break;
-          case 'flip':
-          case 'drawer':
-            sizeDrawer();
-            break;
-        }
-      }
-    });
-  };
-
   formOverride = function () {
     $formContainer.submit(function(e) {
```

Comment:
Can you change to use .on('submit', function(evt) to stay current with the styleguide?